.github/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+# Inspired by: https://github.com/dependabot/dependabot-core/blob/main/.github/dependabot.yml
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "16:00"
```

Comment:
I think saturday morning would give us more time to implement the changes. After all we will have more time in general on weekends vs on weekdays

Reply:
OK done.